### PR TITLE
Add l10n support

### PR DESCRIPTION
**Pull request by irina060981**
For the https://github.com/alpheios-project/alignment-editor-new/issues/15

I added l10n messages for components and errors printed to console.

@balmas, I could also add l10n support for languages in available languages list. Do we need it?

### CODE REVIEW

File: src/lib/controllers/download-controller.js [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
 export default class DownloadController {
-  static download (downloadType, data) {
+  static download (downloadType, data, l10n) {
     switch (downloadType) {
       case 'plainSourceDownload':
-        return this.plainSourceDownload(data)
+        return this.plainSourceDownload(data, l10n)
       default:
-        console.error(`Download type ${downloadType} is not defined.`)
+        if (l10n) {
```
Comment (balmas) [inline src/lib/controllers/download-controller.js:9]: why do we need this test? If there is no language bundle shouldn't we just fail? It seems like a lot of unnecessary work to duplicate all the messages

File: src/lib/controllers/app-controller.js [hunk 2]
```diff
@@ -1,16 +1,24 @@
 import App from '@/vue/app.vue'
 import Vue from '@vue-runtime'
+
 import TextsController from '@/lib/controllers/texts-controller.js'
 
+import L10n from '@/lib/l10n/l10n.js'
```
Comment (balmas) [inline src/lib/controllers/app-controller.js:6]: hmm. can't we import it from alpheios-core?

Comment (irina060981) [reply src/lib/controllers/app-controller.js:6]: We would need to import the whole alpheios-core with its all dependencies - and it is really a huge amount of files.
I think that this module should be a separate package that could be used in any repo without importing alpheios-core.

But while it is inside alpheios-core I simply copied these three files from l10n package.
do you think that it is worth to import the whole alpheios-core with all its dependencies?

File: src/lib/controllers/download-controller.js (hunk 1 above)
Comment (irina060981) [reply src/lib/controllers/download-controller.js:9]: I am not sure but it is for errors to be sure that they would be printed.
But if you think that it is unneccesary - I would remove.

File: src/lib/controllers/app-controller.js (hunk 2 above)
Comment (balmas) [reply src/lib/controllers/app-controller.js:6]: I guess we can wait. We should probably make l10n a distributable package so that we can do this though. I'll enter an issue for that and it can stay as you have it until then. thanks.

File: src/lib/controllers/download-controller.js (hunk 1 above)
Comment (balmas) [reply src/lib/controllers/download-controller.js:9]: let's remove it. I think if we don't have the l10n library the code should fail.

Comment (irina060981) [reply src/lib/controllers/download-controller.js:9]: done

**Comment by irina060981** [comment]
I decided to apply l10n to languages list too.
